app/controllers/vaccinations_controller.rb

```diff
@@ -90,13 +91,26 @@ def set_patient
     @patient = Patient.find(params[:id])
   end
 
-  def set_patient_outcomes
-    @patient_outcomes =
+  def set_patient_consents_triages_and_vaccs_records
```

Comment:
Might be worth naming this after the variable it sets:

```suggestion
  def set_patient_details
```